### ISSUE TITLE
FTX createOrder - Missing Parameter Price

### ISSUE THREAD

**Issue by alsopgroup** [ccxt/ccxt#6218]
I have a custom function that is calling createOrder with FX and I keep getting an error that the paramerer price is missing.

- OS:
- Programming Language version:
- CCXT version:  1.20.20
- Exchange: FTX
- Method: createOrder

Custom function:
 async marketOrder(symbol, amount, side) {
        const type = 'market';
        const price = undefined;
        const params = {};

        return await this.ftx.createOrder(symbol.toUpperCase(),type,side,amount,price,params).then(orderInfo => this.remapOrder(orderInfo.result.id));
    }

Message returned:
`marketOrder FAILED: InvalidOrder: ftx {"error":"Missing parameter price","success":false}`

**Comment by kroitor** [comment]
Hi! Can you plz post your verbose request/response for this request?

```JavaScript
await exchange.loadMarkets ()
exchange.verbose = true // ←--------- add this after loadMarkets, before your createOrder call
return await this.ftx.createOrder(...
```
?

**Comment by alsopgroup** [comment]
handleRestResponse:
 ftx POST https://ftx.com/api/orders 400 BAD REQUEST
Response:
 { 'Account-Id': '1620479',
  'Cache-Control': 'private, max-age=0, no-cache',
  'Cf-Cache-Status': 'DYNAMIC',
  'Cf-Ray': '53f092957eee93b2-SJC',
  Connection: 'close',
  'Content-Length': '52',
  'Content-Type': 'application/json',
  Date: 'Mon, 02 Dec 2019 22:00:15 GMT',
  'Expect-Ct': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"',
  Server: 'cloudflare',
  'Set-Cookie': '__cfduid=dd50e3e3501bbf6ad6a3bf850f4c0e17f1575324014; expires=Wed, 01-Jan-20 22:00:14 GMT; path=/; domain=.ftx.com; HttpOnly; Secure',
  'Strict-Transport-Security': 'max-age=31536000; includeSubDomains; preload',
  'X-Content-Type-Options': 'nosniff' }
 {"error":"Missing parameter price","success":false}

**Comment by kroitor** [comment]
@alsopgroup unfortunately, you've missed the request and copied just the response... the request is a bit more important in this case...

**Comment by kroitor** [comment]
@alsopgroup does it work for you this way:

```JavaScript
async marketOrder(symbol, amount, side) {
        const type = 'market';
        const price = undefined;
        const params = { price: null }; // ←------------- ADD EXPLICIT OVERRIDE HERE
        return await this.ftx.createOrder(symbol.toUpperCase(),type,side,amount,price,params).then(orderInfo => this.remapOrder(orderInfo.result.id));
    }
```

?

**Comment by alsopgroup** [comment]
fetch:
 ftx POST https://ftx.com/api/orders
Request:
 { 'FTX-KEY': 'KEY_REMOVED',
  'FTX-TS': '1575324014901',
  'Content-Type': 'application/json',
  'FTX-SIGN': 'SIGNATURE_REMOVED' }
 {"market":"BTC-PERP","side":"buy","type":"market","size":10}

**Comment by kroitor** [comment]
@alsopgroup ok, thx, let me know if the snippet above works for you.

**Comment by alsopgroup** [comment]
Bingo!  Thanks.

**Comment by kroitor** [comment]
@alsopgroup ok, one moment, we will upload a fix.

**Comment by kroitor** [comment]
@alsopgroup Ok, this issue should be fixed in version 1.20.25 (the new build will arrive in 15 minutes). So, your original code without the override should work from there. We will be happy if you let us know, whether it works or not.

**Comment by alsopgroup** [comment]
Confirmed working on new version
